pkg/ccl/backupccl/backup_test.go

```diff
@@ -1851,285 +1851,6 @@ func TestBackupRestoreControlJob(t *testing.T) {
 	})
 }
 
-func TestRestoreFailCleansUpTTLSchedules(t *testing.T) {
```

Comment:
@otan if you can sign off on this test, thank you!

Reply:
lgtm